logic/requirements.py

```diff
@@ -336,6 +334,94 @@ def __init__(self, options):
+            "YELLOW_STALFOS":    self.attack_skeleton,
+            "SPIKED_BEETLE":     AND(SHIELD, self.attack_hookshot_powder),
+            "THREE_OF_A_KIND":   OR(self.attack_hookshot_no_bomb, SHIELD),
+            "MASK_MIMIC":        OR(self.rear_attack, self.rear_attack_range),
+            "BOO_BUDDY":         OR(BOW, MAGIC_ROD),                                # assumes no torch is nearby
+            "SHROUDED_STALFOS":  self.attack_hookshot_powder,
+            "SWORD_STALFOS":     self.attack_hookshot_powder,
+            "POLS_VOICE":        OR(BOMB, MAGIC_ROD, AND(OCARINA, SONG1)),          # BOW works, but isn't as reliable as it needs 4 arrows.
+            "RED_ZOL":           self.attack_hookshot_powder,                       # this one can split into 2 gels when hit
+            "GREEN_ZOL":         self.attack_hookshot_powder,                       # Zol hiding in the ground, does not split
+            "GEL":               self.attack_hookshot_powder,
+            "BOMBITE":           self.attack_hookshot_powder,                       # the bouncy ones, not to be confused with TIMER_BOMBITE
+            "TIMER_BOMBITE":     OR(SWORD, BOMB, SHIELD),
+            "PAIRODD":           self.attack_no_boomerang,
+            "IRON_MASK":         self.attack_hookshot_powder,
+            "MASTER_STALFOS":    AND(SWORD, BOMB),
+            "STAR":              self.attack_hookshot_powder,
+            "WIZROBE":           OR(BOMB, MAGIC_ROD),                               # BOW works, but isn't as reliable as it needs 4 arrows.
+            "LIKE_LIKE":         self.attack_hookshot_powder,
+            "VIRE":              self.attack_hookshot_no_bomb,
+            "ROPE":              self.attack_hookshot_powder,
+            "GIBDO":             self.attack_gibdos,
+            "ARM_MIMIC":         SWORD,
+            "GREEN_CAMO_MOBLIN": self.attack_hookshot_powder,
+            "RED_CAMO_MOBLIN":   self.attack_hookshot_powder,
+            "BLUE_CAMO_MOBLIN":  self.attack_hookshot_powder,
+            #"GREEN_ORB_MONSTER": AND(self.attack_hookshot, POWER_BRACELET),           # can not be killed on their own
+            #"RED_ORB_MONSTER":   AND(self.attack_hookshot, POWER_BRACELET),           # can not be killed on their own
+            #"BLUE_ORB_MONSTER":  AND(self.attack_hookshot, POWER_BRACELET),           # can not be killed on their own
+            "SEA_URCHIN":        self.attack_hookshot,                              # does not have powder in overworld logic file as option
+            "GIANT_GOPONGA_FLOWER": OR(BOWWOW, HOOKSHOT, MAGIC_ROD, BOOMERANG),
+            "GOPONGA_FLOWER": OR(BOWWOW, HOOKSHOT, MAGIC_ROD, BOOMERANG),
+            "SWORD_MOBLIN":      self.attack_hookshot_powder,
+            "MOBLIN":            self.attack_hookshot_powder,
+            "MAD_BOMBER":        OR(SWORD, BOW, MAGIC_ROD),
+            "CROW":              self.attack_hookshot_no_bomb,
```

Comment:
I see both DACTO and CROW?

Reply:
yea, some things were duplicate as I wasn't sure how the game handled it. When cross referencing with the disassembly I also noticed that every moblin variant is just a moblin or sword moblin